app/controllers/bundle_context_controller.rb

```diff
@@ -1,11 +1,30 @@
 class BundleContextController < ApplicationController
-
   def index
+    @bundle_context = BundleContext.new
```

Comment:
is there a reason why `BundleContext` is instantiated here? it doesn't look like this is used in your `index.html.erb`?

Reply:
yeah, this feels removable, afaict

Reply:
In order to display the errors for specific fields in the `_yaml_form.erb` partial that is used in the index route, the @bundle_context model needed to instantiated in the default view. Removing `@bundle_context = BundleContext.new` breaks the rendering of the form.

Reply:
It makes sense if the template uses the variable, it has to be instantiated here.